operator/pkg/gateway-api/secret_reconcile_test.go

```diff
@@ -154,6 +155,7 @@ func Test_SecretSync_Reconcile(t *testing.T) {
 		Build()
 	r := &secretSyncer{
 		client:           c,
+		logger:           logrus.New(),
```

Comment:
Changed that it's setting `io.Discard` - to avoid unnecessary log output. It's handled like this in other tests too. 

Mapping the `-v` would only work by implementing a custom logger integration - don't think that this is worth the effort with `logrus`.